bodhi_messages/setup.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018  Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+import os
+
+from setuptools import setup, find_packages
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, "README.rst")) as fd:
+    README = fd.read()
+
+
+setup(
+    name="bodhi_messages",
+    version="1.0.0",
+    description="JSON schema for messages sent by Bodhi",
+    long_description=README,
+    url="https://github.com/fedora-infra/bodhi/",
+    # Possible options are at https://pypi.python.org/pypi?%3Aaction=list_classifiers
+    classifiers=[
+        "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
```

Comment:
Is there a universe where I can avoid adding a new Python 2 package you think? Do you think it would be fair for me to say that any packages that want to consume Bodhi's cool new messages need to also use Python 3? @hroncok might be happiest if we can be pure Python 3 here.

I've got an open pull request that drops Python 2 support from the server, FWIW: #2759.

Reply:
I amen, do whatever you want upstream, just don't put the Python 2 part to Fedora ever :)

That said, Python 3 only upstream works best, because nobody is tempted to package it into Fedora as a python2 package :D

Reply:
You'll only need Python 2 one or more Bodhi message consumers require it. I'm not sure what all consumes Bodhi messages, but it probably wouldn't hurt to start with Python 3 only.

Reply:
I say let's just force Python 3 for people who want to use this package. That way I don't have to do a big deprecation/removal in the future. People who aren't ready to switch can just keep using the fedmsg relay.

Reply:
I'm going to edit this commit to make it be Python 3 only.